lib/KTable/index.vue

```diff
@@ -0,0 +1,480 @@
+            }
+            break;
+          case 'ArrowRight':
+            if (colIndex === totalCols - 1) {
+              if (rowIndex === totalRows - 1) {
+                nextColIndex = 0;
+                nextRowIndex = -1;
+              } else {
+                nextColIndex = 0;
+                nextRowIndex = rowIndex + 1;
+              }
+            } else {
+              nextColIndex = colIndex + 1;
+            }
+            break;
+          case 'Enter':
+            if (rowIndex === -1 && this.sortable) {
+              this.handleSort(colIndex);
+            }
+            break;
+          case 'Tab':
+            if (focusableElements.length > 0) {
+              if (!event.shiftKey) {
+                if (focusedElementIndex < focusableElements.length - 1) {
+                  focusableElements[focusedElementIndex + 1].focus();
+                  event.preventDefault();
+                  return;
+                } else {
+                  if (colIndex < totalCols - 1) {
+                    nextColIndex = colIndex + 1;
+                  } else if (rowIndex < totalRows - 1) {
+                    nextColIndex = 0;
+                    nextRowIndex = rowIndex + 1;
+                  } else {
+                    nextColIndex = 0;
+                    nextRowIndex = -1;
+                  }
+                }
+              } else {
+                if (focusedElementIndex < focusableElements.length - 1) {
+                  focusableElements[focusedElementIndex + 1].focus();
+                  event.preventDefault();
+                  return;
+                } else {
+                  if (colIndex > 0) {
+                    nextColIndex = colIndex - 1;
+                  } else if (rowIndex > 0) {
+                    nextColIndex = totalCols - 1;
+                    nextRowIndex = rowIndex - 1;
+                  } else {
+                    nextColIndex = totalCols - 1;
+                    nextRowIndex = totalRows - 1;
+                  }
+                }
+              }
+            } else {
+              if (!event.shiftKey) {
+                if (colIndex < totalCols - 1) {
+                  nextColIndex = colIndex + 1;
+                } else if (rowIndex < totalRows - 1) {
+                  nextColIndex = 0;
+                  nextRowIndex = rowIndex + 1;
+                } else {
+                  nextColIndex = 0;
+                  nextRowIndex = -1;
+                }
+              } else {
+                if (colIndex > 0) {
+                  nextColIndex = colIndex - 1;
+                } else if (rowIndex > 0) {
+                  nextColIndex = totalCols - 1;
+                  nextRowIndex = rowIndex - 1;
+                } else {
+                  nextColIndex = totalCols - 1;
+                  nextRowIndex = totalRows - 1;
+                }
+              }
+            }
+            this.focusCell(nextRowIndex, nextColIndex);
```

Comment:
I have fixed the duplicate call issue, still thinking about how we can re-structure the handler in a simpler way. There are a lot of interdependent functions being executed along with the handler, so it might be a task to figure out an appropriate refactor. Still working on it.

Reply:
Added to Follow-Up issues.